app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -146,3 +166,16 @@ input.search-bar {
     opacity: 0.8;
   }
 }
+
+.form-switch .form-check-input:focus {
```

Comment:
I'm just worried about stepping on Bootstrap's toes (if he got some 😅) https://getbootstrap.com/docs/5.0/forms/checks-radios/

Reply:
Yeah changing the switch default color is super annoying for some reason.

Reply:
We shouldn't be using raw bootstrap in GL3.
Those `form-switch` will be refactored to use `react-bootstrap` [CHECK HERE](https://react-bootstrap.netlify.app/forms/checks-radios/#switches).

Reply:
What are you linking to? It's the only way to remove the blue color om switches as far as I know. 

I don't think it's a big deal to overwrite bootstrap components colors. I would not overwrite paddings, widths, etc.